### ISSUE TITLE
opencv dnn net.forward east 

### ISSUE THREAD

**Issue by SpringRainLu** [opencv/opencv#15979]
##### System information (version)

- OpenCV =>4.12
- Operating System / Platform => Windows 64 Bit
- Compiler => PyCharm 2018 CE

##### Detailed description

i tried to run text detection.py based on my own east model, but it failed at 
''outs = net.forward(outNames)''

cv2.error: OpenCV(4.1.1) .\opencv-python\opencv\modules\dnn\src\dnn.cpp:525: error: (-2:Unspecified error) 
Can't create layer "resnet_v1_50/conv1/BatchNorm/FusedBatchNormV3" of type "FusedBatchNormV3" in function 'cv::dnn::dnn4_v20190621::LayerData::getLayerInstance'

i saved my model in this:

output_graph = "frozen_east_model_02.pb"
output_graph_def = tf.graph_util.convert_variables_to_constants(sess, sess.graph_def,
                                                              ["feature_fusion/Conv_7/Sigmoid",
                                                              "feature_fusion/concat_3"])
tf.train.write_graph(output_graph_def, ".", output_graph, as_text=False)

i have tried to modify model.py ， nevertheless it did not work.
https://github.com/argman/EAST/blob/master/model.py Line 150

c1_1 = slim.conv2d(tf.concat([g[i-1], f[i]], axis=3), num_outputs[i], 1)
pi2 = 0.5 * np.pi
angle_map = (slim.conv2d(g[3], 1, 1, activation_fn=tf.nn.sigmoid, normalizer_fn=None) - 0.5) * pi2 # angle is between [-45, 45]
F_geometry = tf.concat([geo_map, angle_map], axis=3)

**Comment by alalek** [comment]
What is version of TensorFlow?

**Comment by SpringRainLu** [comment]
> What is version of TensorFlow?

tf 1.15.0

**Comment by dkurt** [comment]
@SpringRainLu, please take a look at https://github.com/opencv/opencv/pull/15986 (WIP for now, we will add some tests soon).

**Comment by SpringRainLu** [comment]
Thanks three thousand! @dkurt 
i retried the code in Tensorflow 1.14, and it ran without errors.

**Comment by mapavia** [comment]
> Thanks three thousand! @dkurt
> i retried the code in Tensorflow 1.14, and it ran without errors.

Hey @SpringRainLu, what code did you actually retry? Just the samples/dnn/text_detection.py or did you retrain your own east model again?

**Comment by dkurt** [comment]
> error: OpenCV(4.1.1)

@mapavia, Please try newer version of OpenCV

**Comment by mapavia** [comment]
I got it working by retraining the my own model.

**Comment by mapavia** [comment]
> > error: OpenCV(4.1.1)
> 
> @mapavia, Please try newer version of OpenCV

I'll try that next time. But I got it working by retraining my model using tensorflow 1.14 instead of 1.15. Thanks.
